src/main/java/org/opensearch/security/securityconf/DynamicConfigFactory.java

```diff
@@ -294,7 +294,7 @@ private static ConfigV7 getConfigV7(SecurityDynamicConfiguration<?> sdc) {
     }
 
     @Override
-    public final boolean isInitialized() {
+    public boolean isInitialized() {
```

Comment:
was this changed purely for testing?

Reply:
yes, this is to mock the method. I think this is a safe change as it's a read method with a single caller.